dissect/target/plugins/os/unix/linux/_os.py

```diff
@@ -34,17 +34,16 @@ def detect(cls, target: Target) -> Filesystem | None:
     @export(property=True)
     def ips(self) -> list[str]:
         """Returns a list of static IP addresses and DHCP lease IP addresses found on the host system."""
-        ips = []
+        ips = set()
 
         for ip_set in self.network_manager.get_config_value("ips"):
             for ip in ip_set:
-                ips.append(ip)
+                ips.add(ip)
```

Comment:
I have no idea what kind of type `ip_set` is here, but can't you just `ips.update(ip_set)`?